design/components/data.tsx

```diff
@@ -1,2 +1,4 @@
 export const RICH_TEXT =
   '<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>';
+
+export const SERVICE = '<P>Research & Design</P>';
```

Comment:
Is this still used? Should not be called `SERVICE`.

Reply:
Done by deleting it and adding text isted